### DISCUSSION TITLE
v-click multi stages

### DISCUSSION THREAD

**Discussion by fguisso** [slidevjs/slidev#1914] [Q&A] [answered]
https://sli.dev/guide/animations#enter-leave

I'm trying to create something like this:

```vue
<div v-click="[1,3,9,10]" />
```

I want to have multiple stages where my elements is enter and leave. I thing that this is possible, like in code blocks, where I can create comenting like this:

```md
```js{1-2|3-4|5-6|all}
...
```

**Comment by kermanx** [comment, marked as answer]
One possible solution:

```md
---
clicks: 10
--

<div v-if="[1,2,9].includes($clicks)" />
```

BTW,
````md
```js{1-2}{3-4}{5-6}
...
````
doesn't work in Slidev.

**Reply by fguisso** [reply]
Very good, thanks.
